Description: delete unused service.[ch] files with restrictive license
 include/service.h and src/service.c files are copyrighted by Microsoft
 with just a 'All Rights Reserved' message as license. Supposedly iperf
 is distributed under NCSA license, but licensing information in this
 file should be changed. In the meantime, and as these files are only used
 for compiling the windows iperf binary, I'm deleting them from the Debian
 version.
Author: Roberto Lumbreras <rover@debian.org>

Index: iperf-2.0.13+dfsg1/src/Makefile.am
===================================================================
--- iperf-2.0.13+dfsg1.orig/src/Makefile.am	2019-01-23 18:25:13.003062080 +0100
+++ iperf-2.0.13+dfsg1/src/Makefile.am	2019-01-23 18:25:12.995062019 +0100
@@ -33,7 +33,6 @@
 		gnu_getopt_long.c \
 	        histogram.c \
 		main.cpp \
-		service.c \
 		sockets.c \
 		stdio.c \
 		tcp_window_size.c \
